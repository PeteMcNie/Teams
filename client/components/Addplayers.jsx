import React from 'react'
import { connect } from 'react-redux'
import { Link } from 'react-router-dom'

import Homebutton from './Homebutton'
import Form from './Form'
// import Loading from './Loading'

const Addplayers = (props) => {
// console.log('addplayers ', props)
  // const { loading } = props

  // if (loading) {
  //   return <Loading />
  // }

  return (
    <div>
      <div>
        <Form />
      </div>
      <div className='link'>
        <Link to={'/createteams'}>Create Teams</Link>
      </div>
      <div>
        <Homebutton />
      </div>
    </div>
  )
}

// function mapStateToProps (state) {
//   return {
//     loading: state.loading
//   }
// }

export default connect()(Addplayers)
