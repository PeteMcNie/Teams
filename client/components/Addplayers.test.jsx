import React from 'react'

import { renderWithRedux } from '../testing/utils'

import Addplayers from './Addplayers'

test('Addplayers component matches snapshot', () => {
  const { container } = renderWithRedux(<Addplayers />)
  expect(container).toMatchSnapshot()
})
