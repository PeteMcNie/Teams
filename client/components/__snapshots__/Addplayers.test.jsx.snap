// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Addplayers component matches snapshot 1`] = `
<div>
  <div>
    <div>
      <div>
        <form>
          <div>
            <button>
              Add Person
            </button>
          </div>
          <div>
            <button
              type="submit"
            >
              Submit
            </button>
          </div>
        </form>
      </div>
    </div>
    <div
      class="link"
    >
      <a
        href="/createteams"
      >
        Create Teams
      </a>
    </div>
    <div>
      <div
        class="link"
      >
        <a
          class="home"
          href="/"
        >
          Home
        </a>
      </div>
    </div>
  </div>
</div>
`;
